articles/how-to-consume-data-from-an-API-using-react/index.md

```diff
@@ -0,0 +1,313 @@
+Application Programming Interface (API), allows developers to access and use data from third parties. Today, there are many free and paid APIs. For example, some APIs provide information about movies, animals, weather, and even financial records. Many developers prefer APIs because it helps save time and resources. The time that could have been spent building a back-end application can be invested elsewhere. In React, using an API allows developers to focus on the front-side application. This means that they have enough time to create a visually appealing website.
+
+The goal of this tutorial is to create a React application that consumes data from an API.
+
+### Prerequisites
+
+To follow along in this tutorial, you must have:
+
+ - Basic knowledge in using React and Bootstrap.
+ 
+ - Node.js installed in your computer.
+
+### Step 1 – Understand the data coming from the API
+
+Before creating any react web application, it's vital to evaluate the type of information presented by the API. Some of the areas to look at include `variables names,` `types,` `dates,` and `arrays.` Data from an API is commonly presented in a `JSON` format. This makes it much easier for it to be consumed in the application.
+
+In this tutorial, we will be using [omdapi](http://www.omdbapi.com/). One needs a valid key to gain access to this API. This key can be generated from the omdapi website and looks something like this `cebd9b53`.
+
+The `key` is included in the `API` request, as shown.
+
+```
+http://www.omdbapi.com/?i=tt3896198&apikey=cebd9b53
+```
```

Comment:
You should provide steps on how and where to acquire the API key from the Website with Screenshots.

Also don't include your API key with the article.